modules/microshift-greenboot-microshift-health-script.adoc

```diff
@@ -51,7 +51,7 @@ The `40_microshift_running_check.sh` health check script only performs validatio
 |`exit 1`
 |===
 
-[id="validation-wait-period"]
+[id="validation-wait-period_{context}"]
 == Validation wait period
 The wait period in each validation is five minutes by default. After the wait period, if the validation has not succeeded, it is declared a failure. This wait period is incrementally increased by the base wait period after each boot in the verification loop.
```

Comment:
Is it worthwhile to include an example of the incremental wait period increase? It would help me, but maybe isn't needed for the audience.